view/frontend/web/instantsearch.js

```diff
@@ -96,6 +95,60 @@ requirejs(['algoliaBundle','Magento_Catalog/js/price-utils'], function(algoliaBu
 					$('.algolia-instant-replaced-content').hide();
 					$('.algolia-instant-selector-results').show();
 				}
+			},
+			routing : {
```

Comment:
Could you write a small comment above the `routing:` explaining what URLs the routing is generating?